control-services/cli/semaphore.py

```diff
@@ -497,7 +497,7 @@ def setup(self):
                     '"environment_id": ' + str(self.envId) + ', '
                     '"alias": "' + name + '", '
                     '"playbook": "' + self.cfg['content_repo']['playbooks']['createClass'] + '", '
-                    '"arguments": "[\\"-e\\", \\"class=##### classSize=##\\"]", '
+                    '"dynamic_vars": "[{\\"name\\":\\"class\\",\\"required\\":true},{\\"name\\":\\"classSize\\",\\"required\\":true}]"' + ', '
```

Comment:
So we're switching from arguments with a strict format to required variables with no required structure?

Reply:
The value checking is implemented in the createClass.yml file.  Previously these values were just placeholders to give the user some context